local/include/etc/confluent/docker/ensure

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
```

Comment:
is this file any different than the version in cp-kafka?

Reply:
Yes. This file is making use of the golang utilities ub for checks. Also, we have removed a few checks which are not in the scope of this image